app/tests/unit/validators/applicant.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { applicantSchema } from '../../../src/validators/applicant';
+
+describe('applicantSchema', () => {
+  it('Fields should only accept string', () => {
```

Comment:
Jest naming should follow an RSpec style: https://en.wikipedia.org/wiki/RSpec#it()
When using `it` the name should be continued as a readable sentence, generally following a `it should`, `it should not`, `it is` pattern

eg: `it should only accept string values`

It's about readability when a test fails, as it will appear like

yourModule
    × should do the thing (3 ms)